### PR TITLE
fixes #20450 - change pagination to patternfly style

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by rabajaj0509** [comment]
LGTM 👍

**Comment by ohadlevy** [comment]
@lzap any idea whats wrong with the tests?

**Comment by lzap** [comment]
Will take a look shortly.

**Comment by lzap** [comment]
Tests finally fixed [test].

**Comment by lzap** [comment]
Please rebase on develop first.

**Comment by ohadlevy** [comment]
@lzap rebased.

File: app/controllers/discovered_hosts_controller.rb [hunk 1]
```diff
@@ -336,4 +336,8 @@ def add_custom_facts
       assign_fact_to_category("discovery_subnet", discovery_subnet)
     end
   end
+
+  def resource_class
+    Host::Discovered
+  end
```
Comment (lzap) [inline app/controllers/discovered_hosts_controller.rb]: This should be out of `private` section.

**Comment by lzap** [comment]
Then rebase please and tests should be green. Well, they randomly fail in core currently.

**Comment by lzap** [comment]
Ping, what's above please, low effort.

**Comment by ohadlevy** [comment]
@lzap rebased.

File: app/controllers/discovered_hosts_controller.rb (hunk 1 above)
Comment (lzap) [reply app/controllers/discovered_hosts_controller.rb]: @ohadlevy you still have this in `private` section, I believe this causes all tests to fail.

**Comment by ohadlevy** [comment]
bump?

**Review by lzap — CHANGES_REQUESTED** [review]
Tests are failing because of encapsulation there.

**Comment by ohadlevy** [comment]
@lzap I've rebased and updated based on your comments, but it seems that the test failures indicate at existing failure in discovery (or its fixtures)? can you please take it over from here?

**Comment by lzap** [comment]
The resource_class breaks all the tests, the controller is a hack around Edit Form and that's the reason - did not dig into details. I don't have time to take over, feel free to pick up this one.

**Comment by ohadlevy** [comment]
@rahulbajaj0509 do you feel like picking this one up? I don't have time to work on this atm.

**Pull request closed without merging**